src/librustc/metadata/common.rs

```diff
@@ -225,10 +226,7 @@ pub struct LinkMeta {
     pub crate_hash: Svh,
 }
 
-pub const tag_closures: uint = 0x95;
-pub const tag_closure: uint = 0x96;
-pub const tag_closure_type: uint = 0x97;
-pub const tag_closure_kind: uint = 0x98;
+// GAP 0x94...0x98
```

Comment:
I'm not sure I follow. Not have a gap? Not comment on the gap? I'm not sure how to avoid the gap except for renumbering everything below, which seems kind of pointless? (Or is there something obvious I'm overlooking?)